src/components/security_manager/test/CMakeLists.txt

```diff
@@ -59,8 +59,14 @@ add_custom_target(generate_certificates
   WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
 )
 
+add_custom_target(copy_certificates
+  COMMAND /bin/bash -c "cp -r client server ${TEST_BIN_DIR}"
```

Comment:
@BSolonenko I suggest you using ${CMAKE_COMMAND} -E copy ... instead and use **add_custom_command** with OUTPUT parameter, since the target is always outdated

Reply:
@mkorniichuk this solution does not work for folders